src/App.tsx

```diff
@@ -27,13 +27,14 @@ import { SourceTargetContextProvider } from './contexts/SourceTargetContextProvi
 import { TransactionContextProvider } from './contexts/TransactionContext';
 import { useConnections } from './hooks/useConnections';
 import Main from './screens/Main';
+import { isEmpty } from 'lodash';
```

Comment:
Is it really needed? Like isn't it just doing `parameter.length === 0`?

Reply:
It is needed as connections is a Json object and that would not work.
What I could do is `Object.keys(parameter).length === 0`. That would work - should I change it?

Reply:
Ah I see. Fair enough, let's keep `isEmpty`.

Reply:
I prefer to avoid 3rd party libraries where I can (and here I can :) )